platform/mds/mds-web/src/main/resources/webapp/partials/widgets/entityInstances.html

```diff
@@ -22,7 +22,7 @@ <h4 class="frame-title">{{selectedEntity.module}} <span ui-if="selectedEntity.mo
             </select>
         </span>
         <div class="btn-group">
-            <button ng-click="importEntityInstances()" type="button" class="btn btn-default" ng-show="showImportButton">
+            <button ng-click="importEntityInstances()" type="button" class="btn btn-default" ng-hide="shouldHideButton()">
                 <span class="glyphicon glyphicon-import"></span>
```

Comment:
showImportButton is needed for custom-ui, we can have editable entity but button should be hidden